src/main/java/build/buildfarm/instance/shard/RedissonCasWorkerMap.java

```diff
@@ -34,14 +34,6 @@
  */
 public class RedissonCasWorkerMap implements CasWorkerMap {
 
-  /**
```

Comment:
Doesn't this being unused now mean the state the comment describes is invariant?

Reply:
The name is forwarded to the Redisson container which is used as part of the key names.  There is no need to additionally store the name outside that container.